src/style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  --container-width: 100%;
```

Comment:
Nice, css variables, cool !

Reply:
sorry,should I delete it? I don't remember when I wrote it there and it is never used)

Reply:
If you don't use it - delete it